packages/did/src/Did.utils.ts

```diff
@@ -455,3 +466,20 @@ export function assembleKeyUri(
   }
   return `${did}#${keyId}`
 }
+
+/**
+ * Helper to produce polkadot-js style enum representations, consisting of an object with a single key-value pair.
+ * The enum variant becomes the object's key (first letter capitalized).
```

Comment:
I was hoping for it to be more versatile than that. I don't know about what and why one would want to grep here and how that would cause issues. Does it make a difference how exactly we transform `ed25519` to `Ed25519` and in which way is the less versatile solution preferable?

Reply:
Ah, I understand! The dumbest thing btw with the capitalisation is that the scale encoding does not even distinguish between 'ed25519' and 'Ed25519', both work just fine - but the string literal for the typescript type is capitalised, so we get a type error with the lower-cased variant.

Reply:
the reason is simply that the enum stuff is autogenerated from the rust code / metadata

Reply:
Yeah, I mean why does their rust code use different case for these strings?